src/doc/rustdoc/src/unstable-features.md

```diff
@@ -455,3 +455,7 @@ Some methodology notes about what rustdoc counts in this metric:
 
 Public items that are not documented can be seen with the built-in `missing_docs` lint. Private
 items that are not documented can be seen with Clippy's `missing_docs_in_private_items` lint.
+
+### `disable-minification`: prevent minification of CSS and JS files
```

Comment:
This should be in the "Command-line arguments" chapter, not the "Unstable features" chapter, if we're stabilizing the flag.